umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md

```diff
@@ -0,0 +1,171 @@
+---
+description: "This article will provide detailed steps on how to migrate a Umbraco 7 Cloud project to Umbraco 8."
+---
+
+# Migrate from Umbraco 7 to Umbraco 8 on Umbraco Cloud
+
+{% hint style="warning" %}
+Due to [a known issue](https://github.com/umbraco/Umbraco-CMS/issues/7914), it is **not possible to directly migrate your Umbraco 7 project to the latest Umbraco 8**.
+
+As a workaround, you can either
+
+* Migrate to **version 8.5** as a first step, and then, post-migration, carry out a normal Umbraco upgrade to the latest version of Umbraco 8, or
```

Comment:
Is this workaround even possible on Umbraco Cloud? How would one go about getting a Umbraco 8.5 project? 

Maybe I am missing something? 🤔

Reply:
Yeah, that's a good observation 🤔 
It should still be possible, you would need to do the migration from 7 to 8.5 locally, then migrate it into a Cloud project running 8.latest afterward.

Reply:
That makes sense 👍